server/src/test/java/com/hedera/block/server/persistence/storage/remove/BlockAsDirRemoverTest.java

```diff
@@ -76,7 +76,7 @@ public void testRemoveNonExistentBlock() throws IOException, ParseException {
         }
 
         // Remove a block that does not exist
-        final BlockRemover blockRemover = new BlockAsDirRemover(testPath, FileUtils.defaultPerms);
+        final BlockRemover blockRemover = new BlockAsDirRemover(testPath, DEFAULT_DIR_PERMISSIONS);
```

Comment:
Same comments as above.

Reply:
Also, my comment is the same as my above response, considered only for the tests isolated.

Reply:
✅ Done.